ui/components/node-renderers/footnote.js

```diff
@@ -9,7 +9,9 @@ export default function Footnote({ docNode }) {
     <span className="clearfix node-footnote" id={docNode.identifier}>
       <span className="node-footnote-content">
         <span className="citation-marker">
-          { docNode.marker }
+          <a href={`#citation-${docNode.marker}`}>
```

Comment:
Let's use `Link` for consistency. It should handle anchors correctly.

Reply:
Added!